Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccticketing/ui/certificateselection/DccTicketingCertificateSelectionFragment.kt

```diff
@@ -5,20 +5,22 @@ import androidx.fragment.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
-import androidx.navigation.fragment.navArgs
+import androidx.navigation.navGraphViewModels
 import de.rki.coronawarnapp.R
+import de.rki.coronawarnapp.dccticketing.ui.shared.DccTicketingSharedViewModel
 
 class DccTicketingCertificateSelectionFragment : Fragment() {
 
-    private val args by navArgs<DccTicketingCertificateSelectionFragmentArgs>()
+    private val dccTicketingSharedViewModel:
+        DccTicketingSharedViewModel by navGraphViewModels(R.id.dcc_ticketing_nav_graph)
```

Comment:
What DccTicketingSharedViewModel is doing ? `QrcodeSharedViewModel` is sharing the data already and it should be passed to `DccTicketingConsentOneViewModel` .

Reply:
"What DccTicketingSharedViewModel is doing?"
![image](https://user-images.githubusercontent.com/46747780/143419893-f5e74a21-76e6-46fe-804a-b7dae4b27c14.png)
(description)

It is used to share the current TransactionContext **between all DccTicketing Fragments**. This approach is easier & more comfortable than using Fragment Args.